k8s/node-mdb-ss.yaml

```diff
@@ -76,12 +81,6 @@ spec:
             port: bdb-port
           initialDelaySeconds: 15
           timeoutSeconds: 10
-#        readinessProbe:
-#          httpGet:
-#            path: /
-#            port: bdb-port
-#          initialDelaySeconds: 15
-#          timeoutSeconds: 10
```

Comment:
We already have liveness probe every 15 secs.
The service is exposed outside only when it is alive.